src/models/thelio-major-r3/external-overview.md

```diff
@@ -0,0 +1,23 @@
+# Thelio Major (External Overview)
+
+## Back overview:
+
+![Back Ports](./img/ports-back.webp)
```

Comment:
Looks like the onboard DisplayPort port is missing a label

Reply:
Fixed in https://github.com/system76/tech-docs/pull/162/commits/ee490964fef0222dd33bd8f52c2c8660f6b420bd.